### ISSUE TITLE
LUA error on Classic ERA

### ISSUE THREAD

**Issue by mimixy** [Rottenbeer/ItemRack#251]
15x ItemRack/ItemRack.lua:1285: calling 'GetName' on bad self (Usage: local name = self:GetName())
[string "=[C]"]: in function `GetName'
[string "@ItemRack/ItemRack.lua"]:1285: in function `func'
[string "@ItemRack/ItemRack.lua"]:1011: in function `OnUpdate'
[string "*ItemRack.xml:7_OnUpdate"]:1: in function <[string "*ItemRack.xml:7_OnUpdate"]:1>

Locals:
(*temporary) = SecureTransferDialog {
 0 = <userdata>
 Button2 = Button {
 }
 MoneyLabel = FontString {
 }
 CautionIcon = Texture {
 }
 Text = FontString {
 }
 Border = Frame {
 }
 Button1 = Button {
 }
}

**Comment by yourapple** [comment]
modify line 1286 to this below:
if MouseIsOver(ItemRackMenuFrame) or IsShiftKeyDown() or (frame and frame.GetName and frame:GetName() and frame:IsVisible() and ItemRack.MenuMouseoverFrames[frame:GetName()]) then

**Comment by Alessandro-Barbieri** [comment]
```
17x ItemRack/ItemRack.lua:1290: calling 'GetName' on bad self (Usage: local name = self:GetName())
[string "=[C]"]: in function `GetName'
[string "@ItemRack/ItemRack.lua"]:1290: in function `func'
[string "@ItemRack/ItemRack.lua"]:1016: in function `OnUpdate'
[string "*ItemRack.xml:7_OnUpdate"]:1: in function <[string "*ItemRack.xml:7_OnUpdate"]:1>

Locals:
(*temporary) = Button {
 0 = <userdata>
 Text = FontString {
 }
 Right = Texture {
 }
 Middle = Texture {
 }
 Left = Texture {
 }
}
```

**Comment by romracer** [comment]
Fixed in #264

**Comment by Rottenbeer** [comment]
Please retest with 4.2
